Purchases/Purchasing/StoreProduct.swift

```diff
@@ -140,11 +160,11 @@ public typealias SK2Product = StoreKit.Product
     @available(iOS 12.0, macCatalyst 13.0, tvOS 12.0, macOS 10.14, watchOS 6.2, *)
     override public var subscriptionGroupIdentifier: String? { underlyingSK1Product.subscriptionGroupIdentifier }
 
-    private lazy var formatter: NumberFormatter = {
+    @objc public override var priceFormatter: NumberFormatter? {
```

Comment:
had to remove `lazy` here because this is a computed var. 
I think an optimal version of this would be to use a private static var, so we can check if it's initialized and only do it once, then listen for storefront changes. 
the public var would be the same, it would just redirect internally to the static var.